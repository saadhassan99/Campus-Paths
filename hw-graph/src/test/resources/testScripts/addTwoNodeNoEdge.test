# Tests adding two nodes to the graph

# Create a graph
CreateGraph two_nodes_graph

# Add two nodes
AddNode two_nodes_graph a
AddNode two_nodes_graph b

# Print the nodes in the graph, the outgoing edges from a, 
# and outgoing edges from b
ListNodes two_nodes_graph
ListChildren two_nodes_graph a
ListChildren two_nodes_graph b