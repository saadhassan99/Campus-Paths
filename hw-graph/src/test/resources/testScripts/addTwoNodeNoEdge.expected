# Tests adding two nodes to the graph

# Create a graph
created graph two_nodes_graph

# Add two nodes
added node a to two_nodes_graph
added node b to two_nodes_graph

# Print the nodes in the graph, the outgoing edges from a, 
# and outgoing edges from b
two_nodes_graph contains: a b
the children of a in two_nodes_graph are:
the children of b in two_nodes_graph are: